src/lib/components/chat/ChatMessage.svelte

```diff
@@ -1,11 +1,12 @@
 <script lang="ts">
 	import { marked } from "marked";
 	import type { Message } from "$lib/types/Message";
-	import { afterUpdate, createEventDispatcher } from "svelte";
+	import { afterUpdate, createEventDispatcher, onMount } from "svelte";
 	import { deepestChild } from "$lib/utils/deepestChild";
 	import { page } from "$app/stores";
-
+	import "../../../styles/latex.css";
```

Comment:
Styles can be added directly in a [sveltekit component](https://learn.svelte.dev/tutorial/styling) if needed using the `<style></style>` markup. 

This is more consistent with the way we do styling elsewhere in the app :grin: